src/pymor/vectorarrays/mpi.py

```diff
@@ -70,6 +70,10 @@ def make_array(cls, subtype, count=0, reserve=0):
                    mpi.call(_MPIVectorArray_make_array,
                             subtype[0], subtype=subtype[1], count=count, reserve=reserve))
 
+    @classmethod
+    def from_data(cls, data, subtype):
+        return cls(subtype[0], subtype[1], mpi.call(_MPIVectorArray_from_data, subtype[0], subtype=subtype[1]))
```

Comment:
Something is wrong here. MPIVectorArray.from_data is called on rank 0 only, so `data` would either have to be a handle to the real data on each rank or it would have to be scattered among the MPI ranks. If you do not have a use-case in mind, I think we could drop this for now.